### PR TITLE
Create thank-you.md

### PR DESCRIPTION
**Pull request by Tomcowan1**
### Trello card

https://trello.com/c/WbNIcjb1/2542-create-new-thank-you-page-for-intentions-survey

### CODE REVIEW

**Comment by Tomcowan1** [comment]
Hi @gemmadallmandfe here's the page (my first creation!) for people who have completed the survey. I've just asked Nicola Beech about what they send (i.e. do they call it a 'survey'? Wil update when I find out. 

Any comment welcome though.

**Comment by Tomcowan1** [comment]
Have spoken to Nicola - wording is now aligned. I haven't shared it with the team... do I need to?

File: app/views/content/thank-you.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  Thank you to candidates who have filled in a Get Into Teaching survey.
+date: "2021-11-01"
+image: "media/images/content/hero-images/0007.jpg"
+backlink: "../../"
+navigation: 40
+lid_pixel_event: "Thank you for letting us know"
+---
+### Thank you for letting us know. 
+
+If you are still interested in teaching, here are [5 steps to get you into the classroom](https://getintoteaching.education.gov.uk/steps-to-become-a-teacher).
```
Comment (gemmadallmandfe) [inline app/views/content/thank-you.md:13]:
```suggestion
If you are still interested in teaching, here are [5 steps to get you into the classroom](/steps-to-become-a-teacher).
```

**Comment by gemmadallmandfe** [comment]
@peteryates How do we suppress this page from appearing in the nav?

**Comment by gemmadallmandfe** [comment]
> Have spoken to Nicola - wording is now aligned. I haven't shared it with the team... do I need to?

Looks good to me in terms of content @Tomcowan1 - don't think it needs sharing with anyone else other than Nicola.

File: app/views/content/thank-you.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Thank you"
+description: |-
+  Thank you to candidates who have filled in a Get Into Teaching survey.
+date: "2021-11-01"
+image: "media/images/content/hero-images/0007.jpg"
+backlink: "../../"
+navigation: 40
```
Comment (peteryates) [inline app/views/content/thank-you.md:8]:
```suggestion
```

**Review by peteryates — COMMENTED** [review]
@gemmadallmandfe - we don't currently have a mechanism for preventing markdown pages from appearing in the sitemap but it should be a two minute job to add it - maybe `exclude_from_sitemap: true` or similar

File: app/views/content/thank-you.md (hunk 2 above)
Comment (gemmadallmandfe) [reply app/views/content/thank-you.md:8]: Thanks @peteryates  - do we need a separate PR for noindexing? I'll raise a ticket

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**